native/test/src/tests/msg_edit.rs

```diff
@@ -204,16 +197,13 @@ async fn edit_image_msg() -> Result<()> {
         "./fixtures/PNG_transparency_demonstration_1.png"
     ))?;
 
+    let draft = user.image_draft(
+        "png_file".to_string(),
+        tmp_png.path().to_string_lossy().to_string(),
+        "image/png".to_string(),
+    );
     timeline
-        .edit_image_message(
-            sent_event_id.to_string(),
-            tmp_png.path().to_string_lossy().to_string(),
-            "png_file".to_string(),
-            "image/png".to_string(),
-            None,
-            None,
-            None,
-        )
+        .edit_message(sent_event_id.to_string(), Box::new(draft))
```

Comment:
this test should also test that the content of the final msg in the timeline is actually as we expect ....

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1175/commits/3d5d21d7ad489c5ce281f350dabaaca38f9b9da6